cmd/clusterctl/test/e2e/e2e_suite_test.go

```diff
@@ -0,0 +1,54 @@
+// +build e2e
+
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func TestE2E(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "ClusterCtl E2E Suite")
+}
+
+const (
+	setupTimeout = 10 * 60
+)
+
+var (
+	ctx          context.Context
+	managerImage string
+)
+
+var _ = BeforeSuite(func() {
+	ctx = context.Background()
+	// Docker image to load into the kind cluster for testing
+	managerImage = os.Getenv("MANAGER_IMAGE")
```

Comment:
It might be a silly question, but how the locally built image gets injected in the kind image?